index.js

```diff
@@ -133,6 +133,24 @@ class ExecutorQueue extends Executor {
         return this.api(config, options);
     }
 
+    /**
+     * Unzip the ZIP of artifacts
+     * @method _unzipArtifacts
+     * @param  {Object}  config           Configuration
+     * @param  {Integer} config.buildId   Unique ID for a build
+     * @return {Promise}
```

Comment:
Don't we need to add `token` to param?
https://github.com/screwdriver-cd/models/blob/9b9d5bb9ca83d9f95cef2b44c9912198d31b65c5/lib/build.js#L682

Reply:
thanks, I missed.